### PR TITLE
Adds a lance to Road home

### PR DESCRIPTION
**Pull request by CrabbytheCrab**
## About The Pull Request

Adds Brick Road a white damage lance for road home

It does 50 white with a 4 attack speed to really force you to use its lance part but still being alright. . When you charge with it you leave a brick road that lasts for 30 seconds that slows down abnormalities by 25%.

## Why It's Good For The Game

Giving road home a second weapon would make her more viable

## Changelog
:cl:
add: Adds brick road
imageadd: added charge states to Homing Instinct as a placeholder
/:cl:

### CODE REVIEW

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Inappropriate. 
Add in a new weapon for Road Home.

**Comment by CrabbytheCrab** [comment]
I might but at the same time homing instinct in its current version isn't that good outside of having a p corp rep

**Comment by CrabbytheCrab** [comment]
did it

**Comment by CrabbytheCrab** [comment]
can this get  a second review now thats it is its own weapon @Kitsunemitsu ?

**Review by Kitsunemitsu — APPROVED** [review]
Looks good to me.

File: code/game/objects/items/ego_weapons/he.dm
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/he.dm:640]:
```suggestion
	addtimer(CALLBACK(src, .proc/Slow), 1 SECONDS, TIMER_LOOP)
```
TIMER_LOOP should be used when something wants to run a proc until it qdel's. Note that this can be combined with TIMER_STOPPABLE and an assigned variable `var a = addtimer(...)` to make a looping conditional with the proc. Old coders warn not to abuse this

File: code/game/objects/items/ego_weapons/he.dm
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/he.dm:654]: As discussed it would be better to simply make this apply a unique status effect that increases move_to_delay as opposed to this stacking, as it can result in massive increases, like `3 -> 5.25(+2.25) -> 9.1875(+3.9375) -> 16.078125(+6.890625) -> 13.828125(-2.25) -> 20.26171875 (+8.12109375 - 3.9375)` which is the move_to_delay of a mob with 3 move_to_delay standing in a tile for 6 seconds. This is a 675% increase.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/ego_weapons/he.dm
Comment (LanceSmites328) [inline code/game/objects/items/ego_weapons/he.dm:1734]:
```suggestion
/obj/effect/golden_road2/proc/Slow()
	for(var/mob/living/simple_animal/hostile/H in get_turf(src))
		if(faction_check(H.faction, faction))
			continue
		H.apply_status_effect(/datum/status_effect/brown_bricks)
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

**Review by LanceSmites328 — APPROVED** [review]

**Comment by TheBronJameOffical** [comment]
The lowered right inhand iconstate name has a semicolon in it making it not show up in game.

**Pull request merged**
